src/runtime/internal/dom.js

```diff
@@ -993,7 +993,7 @@ export { ResizeObserverSingleton };
 /**
  * @returns {void} */
 export function toggle_class(element, name, toggle) {
-	element.classList[toggle ? 'add' : 'remove'](name);
+	element.classList.toggle(name, toggle);
```

Comment:
almost made the mistake of merging it with this, better add a comment
```suggestion
	// The `!!` is required because an `undefined` flag means flipping the current state.
	element.classList.toggle(name, !!toggle);
```